go.mod

```diff
@@ -19,3 +19,5 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/cosmos/cosmos-sdk => github.com/fetchai/cosmos-sdk v0.16.6-0.20210618154541-2685be5ca835
```

Comment:
We just need to pull the latest cosmos-sdk version (the one with the updated tendermint dependency):
```suggestion
replace github.com/cosmos/cosmos-sdk => github.com/fetchai/cosmos-sdk v0.16.6-0.20210630084815-8baea8e357b9
```

We don't have tag yet for it, so we can reference the commit directly. Go can autogenerate this updated version by running from the project root:
```bash
# go get <repo>@<commit hash>
go get github.com/fetchai/cosmos-sdk@8baea8e357b9b0c30ceffc5d7e35b7a7bd37a976
```